.github/workflows/tests.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: tests 
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  build_and_test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: complaint_tracker
+          POSTGRES_DB: complaint_tracker_test
+          POSTGRES_PASSWORD: postgres
+        ports: 
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+      
+      - name: Install dependencies
+        run: bundle install
+
+      - name: Setup test database
+        env:
+          RAILS_ENV: test
+          POSTGRES_HOST: localhost
+          POSTGRES_DB: complaint_tracker_test
+          POSTGRES_USER: complaint_tracker 
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_PORT: ${{ job.services.postgres.ports[5432] }}
+        run: |
+          cp config/database.ci.yml config/database.yml
+          bundle exec rake db:create db:migrate
+
+      - name: Precompile assets
+        run: bundle exec rake assets:precompile
```

Comment:
I think we can skip this step. It's not needed (normally) for tests, and the circleci ruby buildpack will do it on deploy.

Reply:
the request specs seem to require the uswds assets, unfortunately. they're new to me, so i definitely might be setting them up wrong, but if we don't precompile, then the tests choke on 
`@import 'uswds/dist/scss/uswds'`

Reply:
I haven't used request specs much before, so disregard my comment!